openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -65,82 +62,87 @@ class Meta:
         app_label = 'course_overviews'
 
     # IMPORTANT: Bump this whenever you modify this model and/or add a migration.
-    VERSION = 16
+    VERSION = 17
 
     # Cache entry versioning.
-    version = IntegerField()
+    version = models.IntegerField()
```

Comment:
All of these changes are just to be more inline with the preferred way of defining these fields according to django